src/test/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketBuildStatusNotificationsTest.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2018, Nikolas Falco
```

Comment:
No need for copy right on a unit test

Reply:
Hi @casz
the other unit tests does have a copyright, should I remove it anyway ?

Reply:
I see no reason why unit tests have copyrights, they are not distributed and they should fall under the license in this repo.

Reply:
That was a reasoning, for yes. So yes please remove it.